packages/lib/sync/services/index.ts

```diff
@@ -0,0 +1,9 @@
+import { ISyncServices } from "../ISyncService";
+import SendgridService from "./SendgridService";
+
+const services: ISyncServices[] = [
+  //CloseComService, This service gets a special treatment after deciding it shouldn't get the same treatment as Sendgrid
```

Comment:
Close.com does not behave the same as Sendgrid, as the definitions changed to only register contacts within teams, so it was excluded from here and implemented specific Close.com functions in SyncServiceManager.